### PR TITLE
add /spectator-follow command

### PR DESCRIPTION
**Pull request by clifffrey**
This is a bit like /where, except that it takes over your main UI.

### Tested Changes:
- [x] I've tested the changes locally or with people.
- [ ] I've not tested the changes.

### CODE REVIEW

File: commands/where.lua [hunk 1]
```diff
@@ -62,6 +62,31 @@ commands.add_command('where', 'Locates a player', function(cmd)
...
+
+    if not cmd.parameter then
+        return
+    end
+    local target_player = game.get_player(cmd.parameter)
+
+    if not target_player or not validate_player(target_player) then
+        return
+    end
+    player.zoom_to_world(target_player.position, nil, target_player.character)
```
Comment (Ragnarok77-factorio) [inline commands/where.lua:83]: Shouldn't there be a check that character exist ? Will it throw an error if you do this command while the player is dead (no character) ? Or it's fine..?

**Review by Ragnarok77-factorio — CHANGES_REQUESTED** [review]
Unsure about this case, except my comment, rest is ok for me

File: commands/where.lua (hunk 1 above)
Comment (clifffrey) [reply commands/where.lua:83]: `validate_player` verifies that the character exists already

**Review by Ragnarok77-factorio — APPROVED** [review]
nothing else to add

**Pull request merged**